README.md

```diff
@@ -10,9 +10,9 @@
 Build | Status
 -- | --
 Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-msgs/tree/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-msgs/tree/main)
```

Comment:
Can you change the branch in Test coverage as well?

Reply:
Changed in [37f4463](https://github.com/gazebosim/gz-msgs/pull/455/commits/37f44633e73855671c0ad5928cf8172777094df4).